source/onboard/sso-gitlab.rst

```diff
@@ -65,9 +68,11 @@ Step 2: Configure Mattermost for GitLab SSO
 4. The **Discovery Endpoint** for OpenID Connect with GitLab is prepopulated with ``https://gitlab.com/.well-known/openid-configuration``.
 5. Paste the **Application ID** from GitLab as the **Client ID** in Mattermost.
 6. Paste the **Application Secret Key** from GitLab as the **Client Secret** in Mattermost. 
-7. Select **Save**.
+7. Update the ``config.json`` file to specify the scopes selected for the ``GitLabSettings`` property. Changes to this setting require a server restart before taking effect.
```

Comment:
Based on the other feedback above, we may need to call out that `openid` is a required scope at a minimum for the connector to work.

Reply:
Should we provide an example to clarify whether this is a space or comma separated list, or is this obvious enough?